.github/workflows/dockerhub-release.yml

```diff
@@ -0,0 +1,24 @@
+name: DockerHub (Release)
+
+on:
+  push:
+    tags:
+      - v*
```

Comment:
So this workflow runs everytime we push a new `v*` tag? That's nice but that means we cannot really afford messing things up. On several occastion I pushed a tag, realized I forgot something, deleted the tag a pushed it again later for instance.

Reply:
But in such occasion, this would be triggered twice and the latest run would be the latest image deployed to Docker Hub, so even on that occasion this would still be fine.

Reply:
Ah yes. I forgot you could overwrite images on dockerhub.